apps/e2e/src/engine-client/fullSanity.ts

```diff
@@ -437,6 +437,19 @@ async function fullSanity(context: RunContext) {
       );
     }
   });
+
+  const transferQuoteResult = await client.transferQuote({
+    chainId,
+    recipient: subaccountToHex({
```

Comment:
Should we add some sort of abstraction such that `recipient: Subaccount` to remove the need for consumers to call `subaccountToHex`? It is a bit unclear from the `recipient` naming what format the field expects

Reply:
Alternatively, given that we only allow transfers to the same address, potentially just need to pass in the destination subaccount name?

Reply:
Yea, I like this.

Reply:
addressed!